bikeshed/shorthands/oldShorthands.py

```diff
@@ -380,10 +380,17 @@ def biblioReplacer(match):
 
     displayInline = "inline" in modifiers
     displayIndex = "index" in modifiers
-    if displayInline and displayIndex:
-        die(f"Biblio shorthand {match.group(0)} contains *both* 'inline' and 'index', please pick one.")
-    elif displayInline or displayIndex:
-        attrs["data-biblio-display"] = "inline" if displayInline else "index"
+    displayDirect = "direct" in modifiers
+    if (displayInline + displayIndex + displayDirect) > 1:
```

Comment:
I hate that Python bool is technically a subclass of int, but I suppose it has its uses...